parsons/etl/etl.py

```diff
@@ -74,6 +74,29 @@ def rename_column(self, column_name, new_column_name):
 
         return self
 
+    def rename_columns(self, column_map):
+        """
+        Rename multiple columns
+
+        `Args:`
+            column_map: dict
+                A dictionary of columns and new names
```

Comment:
Can you specify here that the old names are keys and the new names are values? ie `{'old_name': 'new_name'}`?

Reply:
Sure things. Resolved.